<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSVimporter</class>
 <widget class="QMainWindow" name="CSVimporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>choose file</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextEdit" name="file_path_textedit">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="locale">
        <locale language="Chinese" country="Taiwan"/>
       </property>
      </widget>
      <widget class="QPushButton" name="choose_file_button">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>打開檔案總管</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="errorLabel">
      <property name="font">
       <font>
        <family>微軟正黑體</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
